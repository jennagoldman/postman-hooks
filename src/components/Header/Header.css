h1 {
  font-weight: 600;
  color: #fc40e6;
  margin: 25px 0 0 0;
}

header {
  border-bottom: 1px solid #fc40e6;
} 