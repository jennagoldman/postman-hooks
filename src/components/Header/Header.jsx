import React from 'react';
import './Header.css';

const Header = () => (
  <header>
    <h1>Postman API Client</h1>
  </header>
);

export default Header;
