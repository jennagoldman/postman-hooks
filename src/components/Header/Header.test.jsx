import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header.jsx';

describe('Header', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
