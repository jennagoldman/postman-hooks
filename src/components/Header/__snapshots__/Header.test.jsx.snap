// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches a snapshot 1`] = `
<header>
  <h1>
    Postman API Client
  </h1>
</header>
`;
